src/app/editors/collection-editor/collection-editor.component.spec.ts

```diff
@@ -46,5 +48,59 @@ describe('CollectionEditorComponent', () => {
     component.editorEventListener({});
     expect(router.navigate).toHaveBeenCalledWith(['editors/content-list']);
   });
-
+  it('#ngOnInit should call ngOnInitl and get user data and content identifier', () => {
+    spyOn(component, 'initialize').and.callFake(() => of(true));
+    component.ngOnInit();
+    expect(component.initialize).toHaveBeenCalled();
+    expect(component.userData).toBeDefined();
+    expect(component.queryParams.identifier).toBeDefined();
+  });
+  it('#ngOnDestroy should call ngOnDestroy and set editorConfig to null', () => {
+    component.ngOnDestroy();
+    expect(component.editorConfig).toBe(null);
+  });
+  it('#getEditorMode should call getEditorMode and get content status', () => {
+    component.content = { status: 'review' };
+    const contentStatus = component.getEditorMode();
+    expect(contentStatus).toBe('review');
+  });
+  it('#editorEventListener() should navigate to content list', () => {
+    const router = TestBed.inject(Router);
+    component.editorEventListener({});
+    expect(router.navigate).toHaveBeenCalledWith(['editors/content-list']);
+  });
+  it('#initialize should call initialize', () => {
```

Comment:
@santoshilimi What is meant by `#initialize should call initialize`.
As suggested above, Please use proper naming convention so easy to understand for other developers by looking at the name